shared/api/instance_state.go

```diff
@@ -220,3 +220,30 @@ type InstanceStateNetworkCounters struct {
 	// Example: 179
 	PacketsDroppedInbound int64 `json:"packets_dropped_inbound" yaml:"packets_dropped_inbound"`
 }
+
+// InstanceStateOSInfo represents the operating system information section of an instance's state.
+//
+// swagger:model
+//
+// API extension: instances_state_os_info.
+type InstanceStateOSInfo struct {
```

Comment:
This change should be in the `shared/api: Add OSInfo to InstanceState` commit.

API extension commits should only include `doc/api-extensions.md` and `internal/version/api.go` changes.